app/Http/Controllers/StatusPageController.php

```diff
@@ -74,7 +74,11 @@ public function showIndex()
                                       ->values();
 
             $numIncidentDays = count($allIncidentDays);
-            $numPages = round($numIncidentDays / $appIncidentDays);
+            if ($appIncidentDays > 0) {
```

Comment:
I think that this can be shortened to:

```php
$numPages = max(round($numIncidentDays / $appIncidentDays), 1);
```

Reply:
Sorry, 0 division will still be stopping any further process.
But, using that change is working
            $numPages = round($numIncidentDays / max($appIncidentDays, 1));

Reply:
Okay, that change looks better. Can you update please?